### DISCUSSION TITLE
Desktop host support

### DISCUSSION THREAD

**Discussion by vipw** [leancodepl/patrol#869] [Q&A] [answered]
Hello,

I'm developing a desktop (windows) application in flutter and have started using patrol for my integration tests. What I'm really missing is the ability to take screenshots.

As a workaround, I've added a [Screenshot](https://pub.dev/packages/screenshot) widget as a child of my MaterialApp. It will then work on all platforms, but I don't like adding widgets to my tree that exist only for testing.

**Comment by bartekpacia** [comment, marked as answer]
Hi @vipw,

We're currently not supporting desktop platforms for _native automation feature_ (custom finders work just fine, though!)
